src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserService.kt

```diff
@@ -58,10 +60,15 @@ class UserService(
     telemetryClient.trackEvent("ExternalUserDisabled", mapOf("username" to user.name, "admin" to authenticationFacade.getUsername()), null)
   }
 
+  suspend fun getAllUsersLastName(): List<UserLastNameDto> =
+    userRepository.findAllBySource().map { UserLastNameDto(it.getUserName(), it.lastName!!) }.toList()
```

Comment:
The lastName field on the `User` entity is nullable. If this is null when converting to the `UserLastNameDto` an NPE will be thrown. Is this handled in the controller to convert to an error status?

Reply:
updated to return empty string if null in db